src/xunit.performance.api/Model/AssemblyModel.cs

```diff
@@ -175,6 +175,143 @@ private static IEnumerable<(string testCaseName, string metric, IEnumerable<doub
     }
 
     [Serializable]
+    [XmlRoot("BenchmarkScenario")]
+    public sealed class BenchmarkScenario
+    {
+        [XmlAttribute("Name")]
+        public string Name { get; set; }
+
+        [XmlAttribute("Namespace")]
+        public string Namespace { get; set; }
+        
+        [XmlArray("Assemblies")]
+        public List<NetcoreAssembly> Assemblies { get; set; }
+
+        public BenchmarkScenario()
+        {
+            Assemblies = new List<NetcoreAssembly>();
```

Comment:
If we do need to track assemblies, we should come up with a different name for this class, because the harness can run on more than just .NET core, so the assemblies are not specific to .NET core.